docs/5.0. Definitions.md

```diff
@@ -57,10 +57,10 @@ consists of a JSON object in which each key:value pair is referred to as a claim
 A Bearer Token is passed from the Authorization Server to the client after successful authentication of the request.
 Section 5.1. of [RFC 6749][RFC-6749] defines it as consisting of:
 *   an `access_token` - this is the JWT used to access a protected resource.
-*   an `expires_at` value - this coincides with the lifetime of the access token
+*   an `expires_at` value - this coincides with the lifetime of the Access Token
 *   a `token_type` - always "Bearer" within the scope of this document
-*   a `refresh_token` (OPTIONAL) - this is present depending on the grant type. Refresh tokens can be used to retrieve
-further access tokens from the Authorization Server if the shorter-lived access token has expired.
+*   a `refresh_token` (OPTIONAL) - this is present depending on the grant type. Refresh Tokens can be used to retrieve
```

Comment:
here it's marked optional 🤔 the context is unclear in the diff but I think it's manditory

Reply:
If I remember correctly, it's included for the Authorization Code Grant, but not for the Client Credentials Grant?

Reply:
Ahh, good point, #77 is not merged yes but yes this should stay

Reply:
I've added some extra language to one of the mentions of access tokens in the other documents as I'd forgotten about this case.